src/assemble_workflow/bundle.py

```diff
@@ -109,3 +113,23 @@ def __get_min_bundle(self, build_components):
         if min_bundle is None:
             raise ValueError('Missing min "dist" in input artifacts.')
         return min_bundle
+
+    def copy_file(self, source, dest):
+        file = os.path.realpath(
+            os.path.join(
+                os.path.dirname(os.path.abspath(__file__)),
+                source,
+            )
+        )
+        if not os.path.isfile(file):
```

Comment:
You can't exit in the middle of code. The next call to `shutil.copy2` will fail with an exception as needed.